ceph-build-next/build/build_deb

```diff
@@ -6,6 +6,19 @@ if test -f /etc/redhat-release ; then
     exit 0
 fi
 
+[ "$TEST" = true ] && chacra_ref="$vers" || chacra_ref="test"
+vers=`cat ./dist/version`
+distro=`python -c "exec 'import platform; print platform.linux_distribution()[0].lower()'"`
+chacra_endpoint="ceph/${chacra_ref}/${distro}/{$dist}/${ARCH}"
+
+binary_exists=`chacractl binary exists ${chacra_endpoint}`
+
+# if the binary already exists in chacra, do not rebuild
+if [ "$binary_exists" = true ] ; then
```

Comment:
the command returns `0` for an existing url and `1` for a non existing one, I don't think that just checking the output would be what we want here. We should trap the exit code with `$?`

    if [ $? -e 0 ]; then
        echo "The endpoint at ${chacra_endpoint} already exists and FORCE_REBUILD was not used, Exiting..."
    fi